.github/workflows/update-external-docs.yml

```
@@ -0,0 +1,30 @@
+name: Update External Docs
+on:
+    schedule:
+    - cron:  '0 8 * * 1'
+
+
+permissions:
+    pull-requests: write
+    contents: write
+
+jobs:
+    update-docs:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
+            - name: Run scripts
+              run: bash ./get-contributing.sh && bash ./get-readmes.sh
+            - name: Create Pull Request
+              uses: peter-evans/create-pull-request@v5
+              with: 
+                token: ${{ secrets.ACTIONS_BOT_TOKEN }}
```

```suggestion
                token: ${{ secrets.GITHUB_TOKEN }}
```

Is the automatic token an option in this case? https://docs.github.com/en/actions/security-guides/automatic-token-authentication

---

Yes, it is the default option for the token, and now that we know it is created automatically, it is a good idea to leave it as you suggested.